packages/fxa-auth-server/lib/routes/subscriptions.ts

```diff
@@ -153,7 +118,7 @@ class DirectStripeRoutes {
       );
       for (const key of capabilityKeys) {
         const clientId = key.split(':')[1];
-        const capabilities = splitCapabilities(metadata[key]);
+        const capabilities = splitCapabilities((metadata as any)[key]);
```

Comment:
Not sure it's much better, but might be able to use [this RawMetadata type](https://github.com/mozilla/fxa/blob/main/packages/fxa-shared/subscriptions/types.ts#L3)? I think I spent a chunk of time months ago trying to see if I could cobble together a type around capability strings, but seems like it's more a runtime validation thing

Reply:
Yea, I tried at least 4 different ways to type this, and all of them failed in various sad ways.